.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ '1.20', '1.19', '1.18', '1.17' ]
+        go: [ '1.20', '1.19' ]
```

Comment:
Should we also upgrade the min version of go in `go.mod` to 1.19, if we are not testing against older go version anymore?

Reply:
nvm. looks like you already addressed that below.